Makefile

```diff
@@ -18,18 +18,19 @@ else
   GIT_TREE_STATE=dirty
 endif
 
+SHELL = bash
```

Comment:
We use `[[` somewhere in this Makefile that has been failing in CI and on my dev machine because the default GNU `make` shell is `sh`.